src/prefect/deployments/schedules.py

```diff
@@ -28,6 +29,11 @@ def normalize_to_minimal_deployment_schedules(
                 normalized.append(create_minimal_deployment_schedule(**obj))
             elif isinstance(obj, MinimalDeploymentSchedule):
                 normalized.append(obj)
+            elif isinstance(obj, get_args(SERVER_SCHEDULE_TYPES)):
```

Comment:
Guarding against using the server schemas instead of the client schemas as expected.